Proposals/TypeNetwork_ParametricAxes/ProposalSummary_ytuc.md

```diff
@@ -48,6 +48,9 @@ attribute that all users can easily point to, but rather than calling this axes
 we see conventionality benefits in a name that fits into the systematic structure of the overall
 system of proposed axes: Y dimension transparency of upper case letters.
 
+ytuc is measured from the baseline to the top of Uppercase
```

Comment:
Since this is part of the portion that would be published in the registry, can you please formulate as a full, punctuated sentence; e.g.,

"The ytuc value is measured from the baseline to the top of the uppercase (cap) height."

Also, wouldn't this make sense to be added to the Scale Interpretation field, rather than Additional Information?

Reply:
Waiting on response to the suggested changes.

Reply:
Agreed